internal/terraformtest/eval_context.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) HashiCorp, Inc.
 // SPDX-License-Identifier: BUSL-1.1
 
-package moduletest
+package terraformtest
```

Comment:
I'm not sure I'm happy having a `moduletest` and a `terraformtest` package floating around together. Definitely leads to the question of "what is the difference between them". Perhaps this new package could go into `moduletest/graph` or something?

Reply:
That's a good point. The new package suggestion makes sense. Thanks